Docs/Migration-INI.md

```diff
@@ -0,0 +1,972 @@
+# Migrating from older versions - INI configuration
+
+Migrating to client version [2.11.0.0][client].
+
+This guide uses [YR mod base][mod_base] (commit `34efc04`) configuration
+as an example. The majority of changes also applies to non-YR client
+configurations.
+
+It is **highly recommended** to make a complete backup of your game/mod
+before starting.
+
+## Edit `ClientDefinitions.ini`
+
+The way the client is launched on Unix systems has changed.
+
+1. Add `[Settings]->UnixLauncherExe=Launcher.sh` (script file name can be
+   anything)
+2. Create `Launcher.sh` in root directory:
+
+```sh
+#!/bin/sh
+
+cd "$(dirname "$0")"
+dotnet Resources/Binaries/UniversalGL/clientogl.dll "$@"

```

Comment:
universal gl is net8 exclusive

Reply:
What build should Unix users use with .NET Framework then?

Reply:
.net framework is for windows only. Advanced linux users may explore the possibility with Mono but this is definitely beyond the scope.

For most linux users they should use .net 8 client

Reply:
AFAIK most Linux users currently run the client with .NET 4.0 and XNA through Wine. That should hopefully be possible with .NET 4.8 too.